### PR TITLE
Add Infomaniak Swiss Backup to backend

### PR DESCRIPTION
**Pull request by PAPAMICA**
Simply adding Infomaniak SwissBackup as a backend in OpenStack Swift and S3 Compatible.

Infomaniak SwissBackup website : https://www.infomaniak.com/fr/swiss-backup

### CODE REVIEW

File: Duplicati/Library/Backend/OpenStack/OpenStackStorage.cs [hunk 1]
```diff
@@ -68,6 +68,7 @@ public class OpenStackStorage : IBackend, IStreamingBackend
             new KeyValuePair<string, string>("OVH Cloud Storage", "https://auth.cloud.ovh.net/v3"),
             new KeyValuePair<string, string>("Selectel Cloud Storage", "https://auth.selcdn.ru"),
             new KeyValuePair<string, string>("Memset Cloud Storage", "https://auth.storage.memset.com"),
+            new KeyValuePair<string, string>("Infomaniak SwissBackup", "https://swiss-backup02.infomaniak.com/identity/v3"),
```
Comment (warwickmm) [inline Duplicati/Library/Backend/OpenStack/OpenStackStorage.cs:71]: Thanks @PAPAMICA.  Would `Infomaniak Swiss Backup` (with a space between `Swiss` and `Backup`) be more consistent with the [product name](https://www.infomaniak.com/en/swiss-backup)?

**Comment by warwickmm** [comment]
Sorry, what I was suggesting earlier was to change `SwissBackup` to `Swiss Backup` in the new lines of code.

**Comment by PAPAMICA** [comment]
My bad @warwickmm, I'm still not very comfortable with Github PR ^^ I'll try to change the name as soon as I have time :)

**Comment by kenkendk** [comment]
I have adjusted the names after merging the PR

**Pull request merged**